launcher/src/main.rs

```diff
@@ -501,13 +501,12 @@ fn shard_manager(
 
     if let Some(dtype) = dtype {
         shard_args.push("--dtype".to_string());
-        shard_args.push(dtype.to_string())
+        shard_args.push(dtype.to_string());
```

Comment:
does it fail for you without the semicolon?

Reply:
No it does not fail as 504L is a terminal line . That change is optional though. Let me rephrase the header.